src/App.js

```diff
@@ -1,19 +1,44 @@
-import React from 'react';
-import './App.css';
+import React, { Component } from 'react';
 
+import './App.css';
+import todos from './api/todos';
 import users from './api/users';
 
-function App() {
-  return (
-    <div className="App">
-      <h1>Static list of todos</h1>
+import TodoList from './components/TodoList/TodoList';
+import Form from './components/Form/Form';
+
+class App extends Component {
+  state = {
+    todoList: todos.map(todo => ({
+      ...todo,
+      user: users.find(user => todo.userId === user.id),
+    })),
+  }
+
+  handleAddTodo = (todoFromForm) => {
```

Comment:
не усложняй названия
```suggestion
  handleAddTodo = (todo) => {
```